src/hooks/useArrowKeyNavigation.tsx

```diff
@@ -4,13 +4,21 @@ import { useRef, useEffect } from 'react';
  * A React hook to enable arrow key navigation on a component.
  */
 
-function handleEnter({ event, currentIndex, activeElement }) {
+function handleEnter(
+  { event, currentIndex, activeElement }: { event: KeyboardEvent, currentIndex: number, activeElement: HTMLElement },
+) {
```

Comment:
curious/suggestion: would it improve readability/understandability of these types if these function args and the return data structure were abstracted into an `interface`? E.g., something like the following:

```tsx
interface HandleEnterOptions {
  event: KeyboardEvent;
  currentIndex: number;
  activeElement: HTMLElement;
}

function handleEnter({ event, currentIndex, activeElement }: HandleEnterOptions) { ... }
```

Similar question for `handleArrowKey` below

Reply:
Sure :) a3fea9e291c5b3fc28ccd4be154d5d5695cd8009